frontend/src/components/Menu.js

```diff
@@ -28,9 +29,10 @@ function Menu({
   fixed,
 }) {
   const [shown, updateShown] = useState(false);
-  const [menuPosition, updateMenuPosition] = useState({});
```

Comment:
I think we don't use this anymore? If you remove the whole line you probably can delete a bunch more code here

Reply:
good call. I decided to keep it around, but was still able to remove a bunch of unneeded code here: [e3501d4](https://github.com/HHS/Head-Start-TTADP/pull/2628/commits/e3501d482b4dacd77caba9481a9471342d41896f)